api/api.go

```diff
@@ -0,0 +1,27 @@
+package api
+
+import (
+	"context"
+
+	"github.com/ONSdigital/dp-nlp-search-scrubber/config"
+	"github.com/ONSdigital/dp-nlp-search-scrubber/db"
+	"github.com/gorilla/mux"
+)
+
+// API provides a struct to wrap the api around
+type API struct {
+	Router *mux.Router
+}
+
+// Setup function sets up the api and returns an api
+func Setup(ctx context.Context, r *mux.Router, cfg *config.Config) *API {
+	api := &API{
+		Router: r,
+	}
+
+	dataBase := db.LoadCsvData(ctx, cfg)
+
+	r.HandleFunc("/scrubber/search", FindAllMatchingAreasAndIndustriesHandler(dataBase)).Methods("GET").Name("FindAllMatchingAreasAndIndustriesHandler")
```

Comment:
Please could the path be prefaced with "/v1" - see dp-identity-api for an example. 

Will be same for the others.

Reply:
So that it's easier for me I'll mark the ones I'm done with, with a thumbsup..
Also regarding this conversation is " v1/search?q=somequery" okay?

Reply:
v1/search? isn't ok because that namespace is already used by `dp-search-api`

Reply:
is `v1/scrubber/search?q=` okay?